src/modules/paymentProcessor/PP_Streaming_v1.sol

```diff
@@ -101,13 +101,14 @@ contract PP_Streaming_v1 is Module_v1, IPP_Streaming_v1 {
     ///         client => paymentReceiver => arrayOfStreamIdsWithPendingPayment(uint[]).
     mapping(address => mapping(address => uint[])) private activeStreams;
 
-    /// @dev    Default start, cliff and end times for new payment orders.
-    uint private defaultStart;
-    uint private defaultCliff;
-    uint private defaultEnd;
+    /// @dev    The flags that this PaymentProcessor uses
+    bytes32 internal constant processor_flags =
+        0x000000000000000000000000000000000000000000000000000000000000000e; // 1110
+    /// @dev    The default values for those flags, in ascending order: start, cliff, end
+    uint[3] private defaultValues;
 
     /// @dev	Storage gap for future upgrades.
-    uint[47] private __gap;
+    uint[46] private __gap;
```

Comment:
Any reason why this was decreased? The `processor_flags` are a constant and thus do not use any storage slots, so we only have the 3-length-array instead of three individual variables, which takes the same storage space, so to me 47 would be the right value here still.
```suggestion
    uint[47] private __gap;
```